src/views/About.vue

```diff
@@ -0,0 +1,20 @@
+<template>
+    <div class="about">
+        <Card>
+            <div style="height: 1200px">
```

Comment:
@Andresws12 Is better apply the height width css here. Is necesary apply in the element?

Reply:
@gonzalo-plaza temporal to show the  layout